### ISSUE TITLE
ContrastiveSampler lacks a seed param, while the docstring states it has one

### ISSUE THREAD

**Issue by martin-gorner** [keras-team/keras-nlp#1481]
Contrastive samples docs here:
https://keras.io/api/keras_nlp/samplers/contrastive_sampler/

The function signature does not have a "seed" arg:
`keras_nlp.samplers.ContrastiveSampler(k=5, alpha=0.6, **kwargs)`

But the docstring states:
seed: int. The random seed. Defaults to None.

Probably just a docstring issue. Since this samples is a variant of BeamSampler, it probably does not need a seed.

**Comment by mattdangerw** [comment]
Yep, just a docs issue. Will fix up.
